### PR TITLE
:hammer: fixing for longevity Longevity tests not running after ha in…

### PR DESCRIPTION
**Pull request by lsrinivas-pure**
…crease test failure

**What this PR does / why we need it**:
Fix for Longevity tests not running after ha increase test failure

**Which issue(s) this PR fixes** (optional)
Closes #PTX-4741

**Special notes for your reviewer**:
Jenkins run : http://jen-endurance.pwx.dev.purestorage.com/job/tp-fa-longevity/34/

### CODE REVIEW

**Comment by lsrinivas-pure** [comment]
> 

Hi Vilas,

After discussing with Vinayak, we though that due to these failures lock is not getting released and hence the next tests are not running. Currently Im testing the fix

File: drivers/backup/auth.go [hunk 1]
```diff
@@ -495,7 +495,8 @@ func GetPxCentralAdminCtx() (context.Context, error) {
 // UpdatePxBackupAdminSecret updating "px-backup-admin-secret" token with
 // "px-central-admin" token
 func UpdatePxBackupAdminSecret() error {
-	pxCentralAdminToken, err := GetPxCentralAdminToken()
+	//pxCentralAdminToken, err := GetPxCentralAdminToken()
```
Comment (vinayakshnd) [inline drivers/backup/auth.go:498]: Remove commented code

File: tests/common.go [hunk 2]
```diff
@@ -41,6 +41,7 @@ import (
 	"github.com/portworx/torpedo/drivers"
 	"github.com/portworx/torpedo/drivers/backup"
 	"github.com/portworx/torpedo/pkg/osutils"
+
```
Comment (vinayakshnd) [inline tests/common.go:44]: Remove unnessassary empty lines.

File: tests/common.go [hunk 3]
```diff
@@ -78,6 +79,7 @@ import (
 	"github.com/portworx/torpedo/pkg/log"
 
 	context1 "context"
+
```
Comment (vinayakshnd) [inline tests/common.go:82]: Same here.

File: tests/common.go [hunk 4]
```diff
@@ -2533,7 +2536,8 @@ func TearDownBackupRestoreAll() {
 			Uid:  BackupLocationUID,
 		},
 	}
-	ctx, err := backup.GetPxCentralAdminCtx()
+	//ctx, err := backup.GetPxCentralAdminCtx()
```
Comment (vinayakshnd) [inline tests/common.go:2539]: Remove comment

File: tests/longevity/longevity_test.go [hunk 5]
```diff
@@ -52,39 +51,39 @@ var _ = BeforeSuite(func() {
 	InitInstance()
 	populateIntervals()
 	populateDisruptiveTriggers()
-	err := backup.UpdatePxBackupAdminSecret()
-	Expect(err).NotTo(HaveOccurred())
+	//err := backup.UpdatePxBackupAdminSecret()
```
Comment (vinayakshnd) [inline tests/longevity/longevity_test.go:54]: Remove code

File: tests/longevity/longevity_test.go [hunk 6]
```diff
@@ -52,39 +51,39 @@ var _ = BeforeSuite(func() {
...
+		CoreChecker:      TriggerCoreChecker,
+		// BackupAllApps:                   TriggerBackupApps,
+		// BackupScheduleAll:               TriggerScheduledBackupAll,
+		// BackupScheduleScale:             TriggerScheduledBackupScale,
+		// BackupSpecificResource:          TriggerBackupSpecificResource,
+		// BackupSpecificResourceOnCluster: TriggerBackupSpecificResourceOnCluster,
+		// TestInspectBackup:               TriggerInspectBackup,
+		// TestInspectRestore:              TriggerInspectRestore,
+		// TestDeleteBackup:                TriggerDeleteBackup,
+		// RestoreNamespace:                TriggerRestoreNamespace,
```
Comment (vinayakshnd) [inline tests/longevity/longevity_test.go:81]: Same here

File: tests/longevity/longevity_test.go [hunk 7]
```diff
@@ -95,11 +94,11 @@ var _ = Describe("{Longevity}", func() {
 
 		TriggerDeployNewApps(&contexts, &triggerEventsChan)
 
-		Step("Setup PX backup", func() {
-			// Set cluster context to cluster where torpedo is running
-			SetClusterContext("")
-			SetupBackup("backup-test")
-		})
+		// Step("Setup PX backup", func() {
```
Comment (vinayakshnd) [inline tests/longevity/longevity_test.go:97]: Same here

**Review by vinayakshnd — CHANGES_REQUESTED** [review]

File: drivers/backup/auth.go (hunk 1 above)
Comment (lsrinivas-pure) [reply drivers/backup/auth.go:498]: Hi Vinayak,
these are px-backup changes and needs some additional changes for it work along with regular longevity.
Hence commented for time being and will be fixed in next sprint

File: tests/common.go (hunk 4 above)
Comment (lsrinivas-pure) [reply tests/common.go:2539]: Hi Vinayak,
these are px-backup changes and needs some additional changes for it work along with regular longevity.
Hence commented for time being and will be fixed in next sprint

File: tests/common.go (hunk 2 above)
Comment (lsrinivas-pure) [reply tests/common.go:44]: this is comming by default after saving the file

File: tests/common.go (hunk 3 above)
Comment (lsrinivas-pure) [reply tests/common.go:82]: this is comming by default after saving the file

File: tests/longevity/longevity_test.go (hunk 5 above)
Comment (lsrinivas-pure) [reply tests/longevity/longevity_test.go:54]: Hi Vinayak,
these are px-backup changes and needs some additional changes for it work along with regular longevity.
Hence commented for time being and will be fixed in next sprint

File: tests/longevity/longevity_test.go (hunk 6 above)
Comment (lsrinivas-pure) [reply tests/longevity/longevity_test.go:81]: Hi Vinayak,
these are px-backup changes and needs some additional changes for it work along with regular longevity.
Hence commented for time being and will be fixed in next sprint

**Review by vinayakshnd — APPROVED** [review]
LGTM

**Pull request merged**